openverse_catalog/dags/data_refresh/dag_factory.py

```diff
@@ -66,6 +72,14 @@
 )
 
 
+def _single_value(cursor):
```

Comment:
Not necessarily now, but this is so generic I'd imagine we could move it out to a util file somewhere. Maybe when we have more of these `PostgresResultOperator`s 🎉

Reply:
Totally agree! I think there might be a few of them, like this single result, row count, `fetchall`, etc.